iib/web/exception_handlers.py

```diff
@@ -0,0 +1,37 @@
+import functools
+
+import flask
+from kombu.exceptions import OperationalError
+
+from iib.exceptions import IIBError
+from iib.web import messaging
+
+
+def catch_rabbitmq_error(request):
+    """
+    Catches RabbitMQ error and raises IIBError instead.
+
+    :param Request request: type of an index image build request
+    """
+
+    def get_decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            try:
+                return func(*args, **kwargs)
+            except OperationalError as e:
+
+                payload = flask.request.get_json()
+                req = request.from_json(payload)
+                req.add_state("failed", f"Scheduling of request failed.")
```

Comment:
This doesn't need to be an f-string and I prefer full sentences for user-facing messages.
```suggestion
                req.add_state("failed", f"The scheduling of the build request failed")
```